index.html

```diff
@@ -66,7 +66,140 @@ <h5>Pay as You Go</h5>
+</head>	</head>
+<body>
+ <header>
+    <a class="logo" href="index.html"><img src="img/karma-logo.svg" alt="karma logo"></a>
+    <nav>
+        <ul class="list">
+            <li class="items"><a class="links" href="#"> Meet Karma</a></li>
+            <li class="items"><a class="links" href="#"> How it Works</a></li>
+            <li class="items"><a class="links" href="#"> Store</a></li>
+            <li class="items"><a class="links" href="#"> Blog</a></li>
+            <li class="items"><a class="links" href="#"> Help</a></li>
+            <li class="items"><a class="links" href="#"> Login</a></li>  
+        </ul> 
+    </nav>
+ </header> 
+
+ <main class="main2">
+
+    <section class="section2">
+        <h1>Order your Karma wifi device today!</h1>
+
+        <form action="#">
+            <div class="fieldset">
+                <div class="what">
+                 <label for="first-name">First name *</label>
+                 <input 
+                    type="text" 
+                    id="first-name"
+                    required
+                 />
+                </div>
+                <div class="what">
+                  <label for="last-name">Last name *</label>
+                  <input 
+                    type="text" 
+                    id="last-name"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="address">
+                <div class="what">
+                  <label for="address1">Address *</label>
+                  <input 
+                    type="text"
+                    id="address1"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="address">
+                <div class="what">
+                  <label for="address2">Address 2</label>
+                  <input 
+                    type="text"
+                    id="address2"
+                  />
+                </div>
+            </div>
+
+            <div class="fieldset">
+                <div class="what">
+                  <label for="city">City *</label>
+                  <select name="city" id="city" required>
+                    <option value="" disabled selected>Select your city...</option>
+                    <option value="london">London</option>
+                    <option value="birmingham">Birmingham</option>
+                    <option value="manchester">Manchester</option>
+                    <option value="bristol">Bristol</option>
+                    <option value="liverpool">Liverpool</option>
+                  </select>
+                </div>
+                <div class="what">
+                  <label for="postcode">Postcode *</label>
+                  <input 
+                    type="text"
+                    id="postcode"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="radio-box">
```

Comment:
This should be a `fieldset` tag instead of a `div`, because it's a group of related form controls (radio buttons)